infra/blueprint-test/pkg/utils/transforms.go

```diff
@@ -40,3 +42,20 @@ func GetResultStrSlice(rs []gjson.Result) []string {
 	}
 	return s
 }
+
+func StringFromTextAndArgs(msgAndArgs ...interface{}) string {
```

Comment:
Since this is only used in gcloud (for now) can we keep it unexported in gcloud? In the future if others need it, we can split it out.

Reply:
Done.